BREAKING_CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Breaking Changes
 
+## 0.1.0
+
+### Remove rules that overlap with formatters from the default set
```

Comment:
Nit: `Remove formatter-conflicting rules from the default rule set`?

Reply:
Sure
```suggestion
### Remove formatter-conflicting rules from the default rule set
```